Utilities/SetupForDevelopment.sh

```diff
@@ -80,29 +80,37 @@ echo -e "Git version $git_version is OK.\n"
 
 cd "${BASH_SOURCE%/*}/.." &&
```

Comment:
For some reason cd "${BASH_SOURCE%/*}/.." didn't work on my mac (using bash). When I ran the script from inside the Utilities directory I get the error: "./SetupForDevelopment.sh: line 82: Utilities/GitSetup/setup-user: No such file or directory"

It does work when running from the directory above Utilities.

Reply:
This second `cd` is not needed, and is broken with relative paths. Removed in next patch.